src/harness/BenchmarkDotNet.Extensions/BenchmarkDotNet.Extensions.csproj

```diff
@@ -4,6 +4,7 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
it's a library, so .NET Standard 2.0 is enough.

I could not use `TargetFramework` (no `s` at the end) because of the referenced `common.props` defines `TargetFrameworks` and it simply does not work if we use just `TargetFramework`

https://github.com/dotnet/performance/blob/88eaa838e90b2ea51622786e958644ba7f7deb90/src/common.props#L8-L12